domains/linstall.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "riotoff"
+    },
+    "record": {
+        "CNAME": "riotoff.github.io"
```

Comment:
404 page, please give a preview of your website.

Reply:
https://riotoff.github.io/linstall